test/hotspot/jtreg/containers/docker/TestMemoryAwareness.java

```diff
@@ -41,8 +41,18 @@
 import jdk.test.lib.containers.docker.DockerTestUtils;
 import jdk.test.lib.process.OutputAnalyzer;
 
+import static jdk.test.lib.Asserts.assertNotNull;
+
 public class TestMemoryAwareness {
     private static final String imageName = Common.imageName("memory");
+    private static String hostMaxMem;
+
+    private static String getHostMaxMemory() throws Exception {
+        DockerRunOptions opts = Common.newOpts(imageName);
+        String goodMem = Common.run(opts).firstMatch("total physical memory: (\\d+)", 1);
+        assertNotNull(goodMem, "no match for 'total physical memory' in trace output");
```

Comment:
I've just realised this will not match (yet) since the relevant log line is added in #9880 .

Reply:
Yes. There is a feature to use dependent PRs, which could be used here. Just target `pr/9880` instead of `master`.

Reply:
I've pushed a small excerpt of the changes from #9880 in 1bf3aa341fd93079289f11e620938e6bed40b382 to this PR in order to add the log lines that the tests depend on.

Reply:
We cannot have this in both PRs, though.

Reply:
Indeed: even without this addition, the other one will merge conflict with this PR (assuming this one is merged first). So I can deal with the duplication when I resolve the conflict.